src/sql-parser/src/parser.rs

```diff
@@ -2693,9 +2693,30 @@ impl<'a> Parser<'a> {
     }
 
     fn parse_pg_connection_option(&mut self) -> Result<PgConfigOption<Raw>, ParserError> {
-        let name = match self.expect_one_of_keywords(&[DETAILS, PUBLICATION])? {
+        let name = match self.expect_one_of_keywords(&[DETAILS, PUBLICATION, TEXT])? {
             DETAILS => PgConfigOptionName::Details,
             PUBLICATION => PgConfigOptionName::Publication,
+            TEXT => {
+                self.expect_keyword(COLUMNS)?;
+
+                let _ = self.consume_token(&Token::Eq);
+
+                let value = self
+                    .parse_option_sequence(Parser::parse_object_name)?
```

Comment:
Is this how we parse normal column expressions? Will it work if someone writes `TEXT COLUMNS ("database"."schema"."table"."column")` which contains four elements?